### DISCUSSION TITLE
TabView: TabPanel manually provided id attribute gets overwritten

### DISCUSSION THREAD

**Discussion by softwaremeisterei** [primefaces/community#1092] [PrimeNG]
Hi,

Due to automation requirements I am trying to set the **id** attribute of p-tabPanels, 
but the value I provide is getting ignored, 
instead a kind of automatic value is generated for the id.

I would appreciate any hint how an automation id could be set otherway or this issue to be fixed asap.

Regards,
Edward

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz reproducer so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity
